### PR TITLE
Adds unit test for checking file existence in repo

### LINKED ISSUE #58: Add unittest to check for specific repo files
This repo should have the following list of files included:
- ./Docker or docker/Docker
- ./docker-compose.yml or ./docker/docker-compose.yml
- ./.env_sample
- ./.gitignore
- ./.travis.yml
- ./.codeclimate.yml
- ./CHANGELOG.md
- ./CODE_OF_CONDUCT.md
- ./CONTRIBUTING.md
- ./.github/ISSUE_TEMPLATE
- ./LICENSE.md
- ./.github/PULL_REQUEST_TEMPLATE
- ./README.md
- ./TROUBLESHOOTING.md
- ./USAGE.md
- ./USE_CASES.md

This PR is only asking for tests of the existence of these files, if the files do not exist when you run the tests - do not worry about the tests not passing. We will identify this and create a new PR for the issue.
Thank you!

### PR DESCRIPTION
**Pull request by micheleorselli**
fixes #58

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=60) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=60) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/php-http-client?pullRequest=60) it.</sub>

File: test/unit/FilesExistTest.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        $this->assertFileExists('./.github/PULL_REQUEST_TEMPLATE');
+        $this->assertFileExists('./README.md');
+        $this->assertFileExists('./TROUBLESHOOTING.md');
+        $this->assertFileExists('./USAGE.md');
+        $this->assertFileExists('./USE_CASES.md');
+
+        $composeExists = file_exists('./docker-compose.yml') || file_exists('./docker/docker-compose.yml');
+        $this->assertTrue($composeExists);
+
+        $dockerExists = file_exists('./Docker') || file_exists('./docker/Docker')
```
Comment (mbernier) [inline test/unit/FilesExistTest.php]: This line is missing a `;` so I am adding it in :) 

```
0.21s$ ../../vendor/bin/phpunit . --bootstrap bootstrap.php --filter test* --coverage-clover clover.xml
PHP Parse error:  syntax error, unexpected '$this' (T_VARIABLE) in /home/travis/build/sendgrid/php-http-client/test/unit/FilesExistTest.php on line 28
```

**Comment by thinkingserious** [comment]
@micheleorselli

We have not been able to merge your Pull Request, but because you are awesome - we wanted to make sure you could still get a SendGrid Hacktoberfest shirt.  

Please go **fill [out our swag form](https://dx.sendgrid.com/swag) before Nov 5th** and we will send the shirt! (We know that you might have tried this before and it didn’t work, sorry about that!)

### **You have till Nov 5th to fill out this form** in order to get the Hacktoberfest shirt!

Thank you for contributing during Hacktoberfest! We hope to see you in the repos soon! Just so you know, we always give away a SendGrid shirt for your first ever non-Hacktoberfest PR that gets merged.

File: test/unit/FilesExistTest.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace SendGrid\Test;
+
+class FilesExistTest extends \PHPUnit_Framework_TestCase
+{
+    public function testFileArePresentInRepo()
+    {
+        $this->assertFileExists('./.gitignore');
```
Comment (mbernier) [inline test/unit/FilesExistTest.php]:
```
There was 1 failure:
1) SendGrid\Test\FilesExistTest::testFileArePresentInRepo
Failed asserting that file "./.gitignore" exists.
/home/travis/build/sendgrid/php-http-client/test/unit/FilesExistTest.php:9
```

I am wondering if PHP is not checking the root directory, but in fact, checking the test file's directory? 
Can you please try outputting the directory here to test and fixing the file paths accordingly?

**Comment by mbernier** [comment]
If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/30min) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

File: test/unit/FilesExistTest.php (hunk 2 above)
Comment (micheleorselli) [reply test/unit/FilesExistTest.php]: yup, will try to do it asap

Comment (mbernier) [reply test/unit/FilesExistTest.php]: Thank you!

**Comment by micheleorselli** [comment]
@mbernier done! Let me know if this fixes the problem

**Comment by mbernier** [comment]
This is great, thanks for making the change!!

**Comment by thinkingserious** [comment]
Hello @micheleorselli,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**